resources/metadata/mtaBuild.yaml

```diff
@@ -0,0 +1,111 @@
+metadata:
+  name: mtaBuild
+  description: Performs an mta build
+  longDescription: |
+    Executes the SAP Multitarget Application Archive Builder to create an mtar archive of the application.
+spec:
+  inputs:
+    params:
+      - name: buildTarget
+        type: string
+        description: "mtaBuildTool 'classic' only: The target platform to which the mtar can be deployed. Valid values: 'CF', 'NEO', 'XSA'."
+        scope:
+        - PARAMETERS
+        - STAGES
+        - STEPS
+        mandatory: false
+        default:
+      - name: mtaBuildTool
+        type: string
+        description: "Tool to use when building the MTA. Valid values: 'classic', 'cloudMbt'."
+        scope:
+        - PARAMETERS
+        - STAGES
+        - STEPS
+        mandatory: false
+        default: cloudMbt
+      - name: mtarName
+        type: string
+        description: "The name of the generated mtar file including its extension."
+        scope:
+        - PARAMETERS
+        - STAGES
+        - STEPS
+        mandatory: false
+        default:
+      - name: mtaJarLocation
+        type: string
+        description: "mtaBuildTool 'classic' only: The location of the SAP Multitarget Application Archive Builder jar file, including file name and extension. If you run on Docker, this must match the location of the jar file in the container as well."
+        scope:
+        - PARAMETERS
+        - STAGES
+        - STEPS
+        mandatory: false
+        default:
+      - name: extensions
+        type: string
+        description: "The path to the extension descriptor file."
+        scope:
+        - PARAMETERS
+        - STAGES
+        - STEPS
+        mandatory: false
+        default:
+      - name: platform
+        type: string
+        description: "mtaBuildTool 'cloudMbt' only: The target platform to which the mtar can be deployed."
```

Comment:
Are there only specific values for 'platform' supported, or is this arbitrary? If only specific, they could be mentioned in the description, otherwise please ignore comment.

Reply:
Only specific values. Basically it is some kind of enumeration. But on the level of the metadata we don't have enums.

Reply:
Minor: I just mean that the description could mention the possible values for `platform`.